prime-angular-frontend/src/app/shared/components/site/overview-container/overview-container.component.ts

```diff
@@ -41,6 +42,7 @@ export class OverviewContainerComponent implements OnInit {
   ) {
     this.routeUtils = new RouteUtils(route, router, SiteRoutes.MODULE_PATH);
     this.businessLicences = [];
+    this.businessLicenceUpdated = false;
   }
 
   public onRoute(routePath: string | (string | number)[]) {
```

Comment:
Just putting this comment at the bottom here.

This may be an existing bug, but file names do not show up on the adjudication overview for business license:
![image](https://user-images.githubusercontent.com/34261366/146440774-253ae5b2-468f-40f9-aebe-3be40a2554ce.png)

Reply:
pushed a fix for this :)